src/bundle/DependencyInjection/Configuration/Parser/Pagination.php

```diff
@@ -40,7 +40,10 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
             ->arrayNode('pagination')
                 ->info('System pagination configuration')
                 ->children()
-                    ->scalarNode('search_limit')->isRequired()->end()
+                    ->scalarNode('search_limit')
+                        ->isRequired()
+                        ->setDeprecated('ezsystems/ezplatform-admin-ui', '2.1', 'The child node "%node%" at path "%path%" is deprecated. Use "search.pagination.limit" instead.')
```

Comment:
This will trigger an error in Symfony 5.0 installations, because this method was extended in 5.1

See:
https://github.com/symfony/symfony/blob/5.0/src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php#L167

I know it trigger a deprecation notice in 5.1, but our composer files still mark 5.0 as valid.

Reply:
Good observation, thanks!
@lserwatka / @adamwojs should I adapt it to make it valid for Symfony 5.0 (as @Steveb-p said it will trigger deprecation notice on Symfony 5.1 though)?